package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "recursive-santa",
   "version": "1.0.0",
-  "description": "",
+  "description": "In the initial commit to this project, you have been provided with files that contain a JavaScript module and some associated tests. Currently the `deliverPresents` function is not implemented recursively causing the tests to fail. Use what we have learned to write a recursive solution to the problem of delivering presents.",
```

Comment:
So the reason this file is changing like this is that you are running `npm init` when you clone your project. Remember that `npm init` is for creating a `package.json` file when you don't have one yet. Since the project you cloned already had one, running `npm init` will step on the existing one and make a bunch of changes to it. Fortunately it didn't break it this time but it could. Just keep that in mind going forward, only `npm init` if there is no `package.json` file.